core/keyserver/keyserver.go

```diff
@@ -268,32 +271,30 @@ func (s *Server) BatchGetUser(ctx context.Context, in *pb.BatchGetUserRequest) (
 	// Lookup log and map info.
 	d, err := s.directories.Read(ctx, in.DirectoryId, false)
 	if err != nil {
-		glog.Errorf("adminstorage.Read(%v): %v", in.DirectoryId, err)
-		return nil, err
+		return nil, logTopLevelErr("BatchGetUser", err)
 	}
 
 	// Fetch latest revision.
 	sth, consistencyProof, err := s.latestLogRootProof(ctx, d, in.GetLastVerifiedTreeSize())
 	if err != nil {
-		return nil, err
+		return nil, logTopLevelErr("BatchGetUser", err)
 	}
 	revision, err := mapRevisionFor(sth)
 	if err != nil {
-		glog.Errorf("latestRevision(log: %v, sth: %v): %v", d.Log.TreeId, sth, err)
-		return nil, err
+		errStr := fmt.Sprintf("BatchGetUser - latestRevision(log: %v, sth: %v)", d.Log.TreeId, sth)
+		return nil, logTopLevelErr(errStr, err)
```

Comment:
optional: could do logTopLevelErr("BatchGetUser", glog.Errorf(...)) for better consistency with other log lines